test_integration_app/main.php

```diff
@@ -0,0 +1,108 @@
+            $this->assertEquals(1, $objUtil->isOk($objEvents->security_notification()));
+            $this->assertEquals(1, $objUtil->isOk($objEvents->transaction()));
+            $this->assertEquals(1, $objUtil->isOk($objEvents->update_account()));
+            $this->assertEquals(1, $objUtil->isOk($objEvents->update_content_comment()));
+            $this->assertEquals(1, $objUtil->isOk($objEvents->update_content_listing()));
+            $this->assertEquals(1, $objUtil->isOk($objEvents->update_content_message()));
+            $this->assertEquals(1, $objUtil->isOk($objEvents->update_content_post()));
+            $this->assertEquals(1, $objUtil->isOk($objEvents->update_content_profile()));
+            $this->assertEquals(1, $objUtil->isOk($objEvents->update_content_review()));
+            $this->assertEquals(1, $objUtil->isOk($objEvents->update_order()));
+            $this->assertEquals(1, $objUtil->isOk($objEvents->update_password()));
+            $this->assertEquals(1, $objUtil->isOk($objEvents->verification()));
+
+            // Decisions API
+            $this->assertEquals(1, $objUtil->isOk($objDecisions->getUserDecisions()));
+            $this->assertEquals(1, $objUtil->isOk($objDecisions->getOrderDecisions()));
+            $this->assertEquals(1, $objUtil->isOk($objDecisions->getContentDecisions()));
+            $this->assertEquals(1, $objUtil->isOk($objDecisions->getSessionDecisions()));
+            $this->assertEquals(1, $objUtil->isOk($objDecisions->getDecisions()));
+            $this->assertEquals(0, $objUtil->isOk($objDecisions->apply_decision_to_user()));
+            $this->assertEquals(0, $objUtil->isOk($objDecisions->apply_decision_to_order()));
+            $this->assertEquals(0, $objUtil->isOk($objDecisions->apply_decision_to_session()));
+            $this->assertEquals(0, $objUtil->isOk($objDecisions->apply_decision_to_content()));
+
+            // Wrokflows API
+            $this->assertEquals(1, $objUtil->isOk($objWorkflows->synchronous_workflows()));
+        
+            // Score API
+            $this->assertEquals(0, $objUtil->isOk($objScore->user_score()));//403
+
+            // Verification API
+            $this->assertEquals(1, $objUtil->isOk($objVerification->Send()));
+            $this->assertEquals(1, $objUtil->isOk($objVerification->resend()));
+            $this->assertEquals(1, $objUtil->isOk($objVerification->check()));
+
+            // PSP Merchant Management API
+            $this->assertEquals(0, $objUtil->isOk($objPSPMerchant->create_merchant()));
+            $this->assertEquals(1, $objUtil->isOk($objPSPMerchant->update_merchant()));
+            $this->assertEquals(1, $objUtil->isOk($objPSPMerchant->get_all_merchants()));
+            $this->assertEquals(1, $objUtil->isOk($objPSPMerchant->get_merchant()));
+
+            print_r("tests are completed");
+        }
+    }
+
+    class Utils 
+    {
+
+        public function isOk($response) {
+            print_r($response);
```

Comment:
we don't need to print responses, we can manually added this when debugging or so